### PR TITLE
3dTproject

### PR DESCRIPTION
**Pull request by birajstha**
addeed `prefix`
Tested with CPAC generated command
```
    3dTproject  -input /ocean/projects/med220004p/bshresth/projects/rbc-runs/output2/working/pipeline_RBCv0/cpac_pipeline_RBCv0_sub-PA001_ses-V1W1/_scan_REST_run-1/func_despiked_template_212/vol0000_trans_merged_masked_despike.nii.gz 
                -mask /ocean/projects/med220004p/bshresth/projects/rbc-runs/output2/working/pipeline_RBCv0/cpac_pipeline_RBCv0_sub-PA001_ses-V1W1/_scan_REST_run-1/align_template_mask_to_template_data_space-template_reg-aCompCor_228/MNI152_T1_1mm_brain_mask_resample_resample.nii.gz 
                -ort /ocean/projects/med220004p/bshresth/projects/rbc-runs/output2/working/pipeline_RBCv0/cpac_pipeline_RBCv0_sub-PA001_ses-V1W1/nuisance_regressors_aCompCor_158/_scan_REST_run-1/build_nuisance_regressors/nuisance_regressors.1D 
                -polort 0 
                -prefix residuals.nii.gz
```

### CODE REVIEW

File: descriptors/afni/3dTproject.json [hunk 1]
```diff
@@ -52,7 +52,11 @@
       "command-line-flag": "-cenmode",
       "description": "'kill' or 'zero' or 'ntrp'. Specifies how censored time points are treated in the output dataset:* mode = zero -- put zero values in their place; output dataset is same length as input* mode = kill -- remove those time points;  output dataset is shorter than input* mode = ntrp -- censored values are replaced by interpolated  neighboring (in time) non-censored values,  before any projections, and then the  analysis proceeds without actual removal  of any time points -- this feature is to  keep the spanish inquisition happy.* the default mode is kill !!!.",
       "optional": true,
-      "value-choices": ["KILL", "ZERO", "NTRP"]
+      "value-choices": [
+        "KILL",
+        "ZERO",
+        "NTRP"
+      ]
```
Comment (nx10) [inline descriptors/afni/3dTproject.json:59]: From the description it seems like these should be lowercase. is that correct?

```suggestion
      "value-choices": [
        "kill",
        "zero",
        "ntrp"
      ]
```

Comment (kaitj) [reply descriptors/afni/3dTproject.json:59]: Looking at the doc [web page](https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dTproject.html), it looks like it should be all capitalized.

Can take a look at the command in the container shortly.

Comment (birajstha) [reply descriptors/afni/3dTproject.json:59]: Both works `all-caps` and `non-caps`.
```
3dTproject -cenmode KILL -input /styx_input/0/vol0000_trans_merged_masked_despike.nii.gz -mask /styx_input/1/MNI152_T1_1mm_brain_mask_resample_resample.nii.gz -ort /styx_input/2/nuisance_regressors.1D -polort 0 -prefix desc-residuals.nii.gz
```
```
3dTproject -cenmode kill -input /styx_input/0/vol0000_trans_merged_masked_despike.nii.gz -mask /styx_input/1/MNI152_T1_1mm_brain_mask_resample_resample.nii.gz -ort /styx_input/2/nuisance_regressors.1D -polort 0 -prefix desc-residuals.nii.gz

```

Comment (nx10) [reply descriptors/afni/3dTproject.json:59]: Seems like they do a case insensitive comparison in the source code:

https://github.com/afni/afni/blob/9b2e0debe8e3020ee7d4a51694877c3c41f7a924/src/3dTproject.c#L1345-L1352

lets go with lowercase so it does not look like it screams KILL at you.

**Pull request merged**